nodejs/migrations/20210614115427-create-user.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      slackUserID: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      slackOrganizationID: {
+        type: Sequelize.STRING
+      },
+      slackWorkspaceID: {
+        type: Sequelize.STRING,
+        allowNull: false,
```

Comment:
I should mention this a bit more clearly in the previous comment but what do you think about the user management in this table for org-wide installation? 

If the app works as an org-wide app, the app does not need to know workspace IDs. Therefore, this column can be nullable if we support org-wide. Also, storing something equivalent to `is_enterprise_install: boolean` in this table may allow the app to more easily handle both single workspace installations and org-wide installations.

Another idea would be to have a bit different table like `OrgUsers` for org-wide installation patterns. But, probably, this way will make the app implementation more complex.

Reply:
We'll definitely want to make the app org-aware, but will handle it in a later PR, as it'll give us a chance to walk through a migration one style from to the other.

Reply:
That makes sense 👍